br/tests/run_group_br_tests.sh

```diff
@@ -25,7 +25,7 @@ groups=(
 	["G02"]="br_full_cluster_restore br_full_index br_incremental_ddl br_pitr_failpoint"
 	["G03"]='br_incompatible_tidb_config br_incremental br_incremental_index br_incremental_only_ddl br_incremental_same_table br_insert_after_restore br_key_locked br_log_test br_move_backup br_mv_index br_other br_partition_add_index br_tidb_placement_policy br_tiflash br_tiflash_conflict'
 	["G04"]='br_range br_replica_read br_restore_TDE_enable br_restore_log_task_enable br_s3 br_shuffle_leader br_shuffle_region br_single_table'
-	["G05"]='br_skip_checksum br_split_region_fail br_systables br_table_filter br_txn br_stats br_clustered_index br_crypter'
+	["G05"]='br_skip_checksum br_split_region_fail br_systables br_table_filter br_txn br_stats br_clustered_index br_crypter br_encryption'
```

Comment:
If the test case run for a long time, you may add a new group for it. (It seems this can be a huge test...)

Reply:
yeah I will add to a new group if it's getting too slow, seems adding new group needs to change config in another repo